src/components/custom/dashboard/client/pages/diet-tracker/MealDetailModal.jsx

```
@@ -2,344 +2,381 @@
+              <span>150%</span>
+              <span>200%</span>
+            </div>
+          </div>
+
+          {/* Plus/Minus Fine Control */}
+          <div className="flex items-center justify-center gap-3 mt-4">
+            <button
+              className="w-10 h-10 rounded-lg bg-gradient-to-br from-zinc-800 to-zinc-900 border border-zinc-700/50 flex items-center justify-center text-white hover:bg-[#FF6B00] hover:border-[#FF6B00]/50 transition-all disabled:opacity-50"
+              onClick={() => setPortion(Math.max(0.25, portion - 0.25))}
+              disabled={portion <= 0.25}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)